src/audits/javascript/without-javascript.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Audit = require('../audit');
+
+class WithoutJavaScript extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'JavaScript',
+      name: 'without-javascript',
+      description: 'Page contains some content when its scripts are not available',
+      requiredArtifacts: ['html', 'htmlWithoutJavaScript']
+    };
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+
+    let bodyHasContent = true;
+    let debugString;
+    if (artifacts.htmlWithoutJavaScript.trim() === "") {
+      bodyHasContent = false;
+      debugString = `The page body should render some content if its scripts are not available.`
+    }
+
+    let text = artifacts.htmlWithoutJavaScript.replace(/\n/g, '').replace(/<.*?>/g, ''); // tony the pony! he comes!
```

Comment:
I don't think we can pull `body.textContent`, because the chrome debugging API doesn't provide that function, and I can't inject JS into the page to do it... because I've disabled JavaScript :) That's why it strips the HTML in the audit, rather than handing back stripped HTML from the gatherer. Maybe I'm wrong, though?